lib/game.rb

```diff
@@ -34,13 +34,15 @@ def output_start
   end
 
   def output_goodbye
+    puts 'See ya later!'
```

Comment:
good switch- you couldn't see the bye after the exit too well and it leaves the terminal with a clean slate

Reply:
i agree! glad we changed it!